Signal.xcodeproj/project.pbxproj

```diff
@@ -61,8 +61,6 @@
 		34330A5C1E787A9800DF2FB9 /* dripicons-v2.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 34330A5B1E787A9800DF2FB9 /* dripicons-v2.ttf */; };
 		34330A5E1E787BD800DF2FB9 /* ElegantIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 34330A5D1E787BD800DF2FB9 /* ElegantIcons.ttf */; };
 		34330AA31E79686200DF2FB9 /* OWSProgressView.m in Sources */ = {isa = PBXBuildFile; fileRef = 34330AA21E79686200DF2FB9 /* OWSProgressView.m */; };
-		34382266209A4E400094FEB7 /* ContactShareApprovalViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 34382265209A4E3F0094FEB7 /* ContactShareApprovalViewController.swift */; };
-		3438226A209B63500094FEB7 /* EditContactShareNameViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 34382269209B63500094FEB7 /* EditContactShareNameViewController.swift */; };
```

Comment:
In the course of this work, I realized that there were a bunch of project issues.

* Many view controllers in `SignalMessaging` weren't in the "ViewControllers" directory.  I've moved them.
* Some views/view controllers in `SignalMessaging` directories were added in the `Signal` project.  I've fixed this.
* `SignalMessaging`'s "project groups" and "directory structure" had many inconsistencies (e.g. some files in directory "A" added in group "A" and others in group "B". I've fixed this.
* `SignalMessaging` had two different "view controllers" groups.  I've fixed this.

Mostly these fixes should just affect the project file(s) and involve moving a few files around within the directory structure.